setup.py

```diff
@@ -19,7 +19,7 @@ def find_stub_files(name):
 with open("README.md", "r") as f:
     readme = f.read()
 
-dependencies = ["mypy>=0.750,<0.780", "django-stubs>=1.3.0", "typing-extensions"]
+dependencies = ["mypy>=0.750", "django-stubs>=1.3.0", "typing-extensions"]
```

Comment:
Let's bump `mypy` to `0.782` and `django-stubs` to `1.6`

Reply:
done